mlem/core/requirements.py

```diff
@@ -92,14 +85,24 @@ class InstallableRequirement(PythonRequirement):
     package_name: Optional[str] = None
     """Pip package name for this module, if it is different from module name"""
 
+    @root_validator
```

Comment:
why not `@validator("pachage_name")`?

Reply:
because you need `module` as well - and it won't be accessible that way